### PR TITLE
`url` Deleted Based on `domain`: The `B`s

### PR DESCRIPTION
**Pull request by johntheholman**
- Removes the `url` key if the `domain` key is the same
- Small updates to some files based on the `editorconfig`
- This is only looking at the `B` files (as seen when hitting the `v3/all.json` endpoint)

### CODE REVIEW

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Please revert the following entries: 
* bochk.com
* bill.com
* boursedirect.fr
* barnesandnoble.com
* mbna.com

**Comment by johntheholman** [comment]
These work:
- bill.com
- barnesandnoble.com
- mbna.com

**Comment by Carlgo11** [comment]
```SH
$ curl https://barnesandnoble.com/
curl: (60) SSL: no alternative certificate subject name matches target host name 'barnesandnoble.com'

$ curl https://bill.com/
curl: (28) Failed to connect to bill.com port 443 after 45003 ms: Connection timed out

$ curl https://mbna.com/
curl: (7) Failed to connect to mbna.com port 443 after 45003 ms: Connection refused
```

**Comment by Carlgo11** [comment]
As the requested changes haven't been resolved yet I'm going to label this PR as inactive.
> **Warning**
> Unless there's activity within the next 7 days I'll close the PR.

**Comment by Carlgo11** [comment]
Only the sites mentioned in https://github.com/2factorauth/twofactorauth/pull/7363#pullrequestreview-1280014747 needs to be reverted.

File: entries/b/bill.com.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "Bill": {
     "domain": "bill.com",
-    "url": "https://www.bill.com",
```
Comment (Carlgo11) [inline entries/b/bill.com.json]: This one too

Comment (johntheholman) [reply entries/b/bill.com.json]: Got that one now. Thanks for keeping me honest. Got distracted while working on this.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**